recipes/go/build.sh

```diff
@@ -0,0 +1,32 @@
+# First, build go1.4 using gcc, then use that go to build go>1.4
+# Should I make this a separate package called go-bootstrap?
+# Should I check the checksum of this tarball? What about windows?
+# https://storage.googleapis.com/golang/go1.4-bootstrap-20170531.tar.gz.sha256
+mkdir go-bootstrap && pushd $_
+curl -L https://storage.googleapis.com/golang/go1.4-bootstrap-20170531.tar.gz | tar -xzf -
+export GOROOT_BOOTSTRAP=$PWD/go
+cd $GOROOT_BOOTSTRAP/src
+./make.bash
+
+pushd $SRC_DIR/src
+if [[ $(uname) == 'Darwin' ]]; then
+  # Tests on macOS are flaky and receive SIGABRT on Travis :-/
+  ./make.bash
```

Comment:
It's deep somewhere in the cgo tests, for Windows some of them have been disabled, it seems. I am not sure what exactly causes this, since they pass locally on my machine.

Reply:
Ok let's add it as an issue on the feedstock repo when this gets out.